apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -67,7 +106,7 @@ export function AiGeneratePage({ resLocals }: { resLocals: Record<string, any> }
               <select id="user-prompt-example" onchange="setPromptToExample()">
```

Comment:
Please use Bootstrap styling here (https://getbootstrap.com/docs/4.6/components/forms/#select-menu).

Reply:
Changed in 09ebd145316f730bdcbac5525887fda484b63483